docs/topics/properties.md

```diff
@@ -41,7 +41,7 @@ The initializer, getter and setter are optional. Property type is optional if it
 (or from the getter return type, as shown below).
 
 ```kotlin
-var allByDefault: Int? // error: explicit initializer required, default getter and setter implied
+var allByDefault // error: explicit initializer required, default getter and setter implied
 var initialized = 1 // has type Int, default getter and setter
```

Comment:
The current example is a bit hard to understand because it starts from broken code and continues with a good one. Let's swap the lines and comment out the incorrect one to separate it from the good code.
```suggestion
var initialized = 1 // has type Int, default getter and setter
// var allByDefault // ERROR: explicit initializer required, default getter and setter implied
```

Reply:
I think your suggestion makes the example clearer.

When I read the example the first time, I thought it was crafted so that I could paste it into my IDE to see an error (see attached screenshot).
![image](https://user-images.githubusercontent.com/3910632/109256721-a1fdeb00-77c4-11eb-82ae-35165b2f1697.png)